### DISCUSSION TITLE
Saved filter permissions?

### DISCUSSION THREAD

**Discussion by FliesLikeABrick** [netbox-community/netbox#11654] [Q&A] [answered]
Hello,

We are excited to use the new "saved filters" feature, however when we try to click on that field in the Filters tab we receive a permissions error "Error Fetching Options for field 'filter_id'\n You do not have permission to perform this action"

What permission needs to be added to our users to access this feature? In the long list of features to add to our user group, I do not see anything relating to saved filters

**Comment by kkthxbye-code** [comment, marked as answer]
> What permission needs to be added to our users to access this feature?

Without testing, this one would be my guess:

<img width="302" alt="Screenshot 2023-02-09 at 21 42 59" src="https://user-images.githubusercontent.com/400797/217934880-53173790-f16b-4231-87be-3e903b1113ab.png">

**Reply by FliesLikeABrick** [reply]
Thanks, for some reason I don't recall this appearing correctly back when I was trying to navigate our changes, but did end up solving it with this.
